drivers/spi/spi_mcux_lpspi.c

```diff
@@ -603,7 +608,7 @@ static int spi_mcux_transceive(const struct device *dev, const struct spi_config
 #ifdef CONFIG_SPI_MCUX_LPSPI_DMA
 	const struct spi_mcux_data *data = dev->data;
 
-	if (data->dma_rx.dma_dev && data->dma_tx.dma_dev) {
+	if (lpspi_inst_has_dma(data)) {
```

Comment:
sorry, I don't understand what did you mean, which commit ?

Reply:
this is just using the helper function created for this check, since this check happens in multiple places, it's easier to read the code with this helper function

Reply:
Here you are checking 
if(data != data)
and the original code checked 
if(rx.dma_dev && tx.dma_dev)

Reply:
im not sure where you are seeing that, what happened here is that I abstracted that expression that previously was the condition into a helper function

Reply:
Never mind this comment, I was looking at line 547